MAUI/Segmented-Control/accessibility.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Segmented Control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion .NET MAUI Segmented Control (SfSegmentedControl).
```

Comment:
Learn here about Accessibility support with the Syncfusion .NET MAUI Segmented Control (SfSegmentedControl).

Reply:
Changed.